plugins/modules/load_balancer_pool_member.py

```diff
@@ -0,0 +1,258 @@
+author:
+  - Gaudenz Steinlin (@gaudenz)
+  - Kenneth Joss (@k-304)
+version_added: "2.3.0"
+options:
+  state:
+    description:
+      - State of the load balancer pool member.
+    choices: [ present, absent ]
+    default: present
+    type: str
+  name:
+    description:
+      - Name of the load balancer pool member.
+      - Either I(name) or I(uuid) are required.
+    type: str
+  uuid:
+    description:
+      - UUID of the load balancer.
+      - Either I(name) or I(uuid) are required.
+    type: str
+  load_balancer_pool:
+    description:
+      - UUID of the load balancer pool.
+    type: str
+  enabled:
+    description:
+      - Pool member will not receive traffic if false. Default is true.
+    default: true
+    type: bool
+  protocol_port:
+    description:
+      - The port to which actual traffic is sent.
+    type: str
+  monitor_port:
+    description:
+      - The port to which health monitor checks are sent.
+      - If not specified, protocol_port will be used. Default is null.
+    default: null
+    type: str
+  address:
+    description:
+      - The IP address to which traffic is sent.
+    type: str
+  subnet:
+    description:
+      - The subnet of the address must be specified here.
+    type: str
+  tags:
+    description:
+      - Tags assosiated with the load balancer. Set this to C({}) to clear any tags.
+    type: dict
+extends_documentation_fragment: cloudscale_ch.cloud.api_parameters
+'''
+
+EXAMPLES = '''
+# Create a pool member for a load balancer pool
+- name: Create a load balancer pool member
+  cloudscale_ch.cloud.load_balancer_pool_member:
+    name: 'my-shiny-swimming-pool-member'
+    load_balancer_pool: '20a7eb11-3e17-4177-b46d-36e13b101d1c'
+    enabled: true
+    protocol_port: '8080'
+    monitor_port: '8081'
+    subnet: '70d282ab-2a01-4abb-ada5-34e56a5a7eee'
+    address: '172.16.0.100'
+    tags:
+      project: ansible-test
+      stage: production
+      sla: 24-7
+    api_token: xxxxxx
+
+# Get load balancer pool member facts by name
+- name: Get facts of a load balancer pool member by name
+  cloudscale_ch.cloud.load_balancer_pool_member:
+    name: 'my-shiny-swimming-pool-member'
+    api_token: xxxxxx
+'''
+
+RETURN = '''
+href:
+  description: API URL to get details about this load balancer
+  returned: success when not state == absent
+  type: str
+  sample: https://api.cloudscale.ch/v1/load-balancers/pools/20a7eb11-3e17-4177-b46d-36e13b101d1c/members/b9991773-857d-47f6-b20b-0a03709529a9
+uuid:
+  description: The unique identifier for this load balancer pool member
+  returned: success
+  type: str
+  sample: cfde831a-4e87-4a75-960f-89b0148aa2cc
+name:
+  description: The display name of the load balancer pool member
+  returned: success
+  type: str
+  sample: web-lb-pool
+enabled:
+  description: THe status of the load balancer pool member
+  returned: success
+  type: str
```

Comment:
Would it be possible to convert this to a boolean?

Reply:
Changed type to bool.